sqlite-schema.sql

```diff
@@ -137,6 +137,33 @@ CREATE INDEX IF NOT EXISTS "ix_substep_id" ON "TimelineSubstep" (
 	"substepId"	ASC
 );
 
+CREATE TABLE IF NOT EXISTS "SubstepData" (
+	"id"	INTEGER NOT NULL,
+	"stepId"	INTEGER NOT NULL,
+	"substepNr"	INTEGER NOT NULL,
+	"dataId"	INTEGER NOT NULL,
+	"relationType"	VARCHAR(50) NOT NULL COLLATE NOCASE,
+	FOREIGN KEY("stepId") REFERENCES "TimelineSubstep"("stepId"),
+	FOREIGN KEY("substepNr") REFERENCES "TimelineSubstep"("substepNr"),
```

Comment:
this probably has to be combined into a single foreign key clause, as it is a composite key